test/skate_web/router_test.exs

```diff
@@ -2,42 +2,28 @@ defmodule SkateWeb.RouterTest do
   use SkateWeb.ConnCase
   use Skate.DataCase
 
-  alias SkateWeb.AuthManager
-
   describe "GET /" do
+    @tag :authenticated
     test "shows you the app", %{conn: conn} do
-      {:ok, token, _} = AuthManager.encode_and_sign("FAKE_UID")
-
-      conn =
-        conn
-        |> put_req_header("authorization", "bearer: " <> token)
-        |> get("/")
+      conn = get(conn, "/")
 
       assert html_response(conn, 200) =~ "div id=\"app\""
     end
   end
 
   describe "GET /search (client-side route)" do
+    @tag :authenticated
```

Comment:
we should have made this a tag a long time ago

Reply:
It's one of the things I copied over from Signs UI and it made our tests there soooo much smoother than they would have been otherwise.

Reply:
Oh wow, this is so great! I'm totally stealing it.